editor/editor_properties.cpp

```diff
@@ -2236,8 +2236,8 @@ EditorPropertyRID::EditorPropertyRID() {
 
 ////////////// RESOURCE //////////////////////
 
-void EditorPropertyResource::_resource_selected(const RES &p_resource) {
-	if (_can_use_sub_inspector(p_resource)) {
+void EditorPropertyResource::_resource_selected(const RES &p_resource, bool p_edit) {
```

Comment:
Same here for the method bind.

Reply:
And in this file as well most `ClassDB::bind_method` calls don't list any properties. 🤔